src/resolvers/Mutation/addUserToUserFamily.ts

```diff
@@ -52,9 +52,9 @@ export const addUserToUserFamily: MutationResolvers["addUserToUserFamily"] =
     //check whether user is admin of the family
     await adminCheck(currentUser?._id, userFamily);
 
-    const isUserMemberOfUserFamily = userFamily.users.some((user) => {
-      user.equals(args.userId);
-    });
+    const isUserMemberOfUserFamily = !!userFamily.users.find((user) =>
+      user.equals(args.userId),
+    );
```

Comment:
Why this change? Aren't they equivalent? The earlier one has better readability IMO.

Reply:
That some method always returns false even if two objectIds are same which is the reason the validation code wasn't tested at all (not covered). @EshaanAgg